README.md

```diff
@@ -17,8 +17,16 @@ I have no idea so far.
 
 ## Installation
 
-There are no other installation requirements for this project.
-It is higly recommended to create a separate virtual enviroment.
+When you clone the project you can use command below for install all needed requirements:
+```
+python -m pip install -r requirements.txt
+```
+
+List of requirements:
+* [html5lib](https://html5lib.readthedocs.io/en/latest/)
+
+
+It is higly recommended to create a separate virtual enviroment for this project.
```

Comment:
Maybe it would be better to put the information about the virtual environment above the installation instructions. And inform about the need to activate it every time you want to work on the project.
And by the way there's a typo in the word _environment_.

Reply:
I leave it at the end because I think that this information will be obviously for most contributors. Topic should be start with the most interesting info for everyone, I think.
Thank you for the typo I fix it.